Dockerfile.ui-builder

```diff
@@ -0,0 +1,28 @@
+ARG image_tag=latest
+FROM elifesciences/pattern-library_assets:${image_tag} AS assets
+FROM elifesciences/php_cli:22434ef5bda09326d4c9347de7d8c2f1610a0b83
+
+USER elife
+ENV PROJECT_FOLDER=/srv/pattern-library-ui
+RUN mkdir ${PROJECT_FOLDER} && \
+    mkdir ${PROJECT_FOLDER}/public
+WORKDIR ${PROJECT_FOLDER}
+COPY --chown=elife:elife \
+    composer.json \
+    composer.lock \
+    ${PROJECT_FOLDER}/
+# customized command for composer
+RUN composer --no-interaction install
```

Comment:
We'll need to extract this when updating the base PHP image.

Reply:
This could use the `composer` image exclusively? I don't see other dependencies